index.js

```diff
@@ -279,7 +279,7 @@ function css (css, file) {
       if (file === 'stdin' && output) file = output
 
       // TODO: Unit test this
-      if (file !== 'stdin') options.from = file
+      options.from = file === 'stdin' ? path.join(process.cwd(), 'stdin.css') : file
```

Comment:
The dummy file `stdin.css` is a requirement or would `process.cwd()` just be fine also ? Sourcemaps won't work either way